src/couch_replicator_scheduler.erl

```diff
@@ -207,14 +207,17 @@ not_recently_crashed(#job{} = Job) ->
     case crash_history(Job) of
         [] ->
             true;
-        [{crashed, When} | _] ->
+        [{{crashed, _Reason}, When} | _] = Crashes ->
+            BackoffExp = erlang:min(1 bsl length(Crashes), ?MAX_BACKOFF_EXPONENT),
```

Comment:
Hrm, `?MAX_BACKOFF_EXPONENT` may be a bit of a misnomer, but the code looks correct to me. Could be `1 bsl erlang:min(length(Crashes), ?MAX_BACKOFF_EXPONENT)`. I'll make that change.